jobs/migrations/insert_export_formats.py

```diff
@@ -11,6 +11,8 @@ class Migration(migrations.Migration):
 
     def insert_export_formats(apps, schema_editor):
         ExportFormat = apps.get_model('jobs', 'ExportFormat')
+        ExportFormat.objects.create(name='MWM Format', description='Maps.me File format',
+                                    slug='MWM')
```

Comment:
this is a problem, because you are changing a migration file, so, if an osm-export-tool2 installation like (http://export.hotosm.org) has already applied migrations, this change cannot be applied, which is a problem

you need to create a new custom migration and add MWM setup to that migration